### DISCUSSION TITLE
Apple Push Notification Service Server Certificate Update

### DISCUSSION THREAD

**Discussion by petrnymsa** [firebase/flutterfire#16842] [Q&A]
We've got email from Apple stating 

As we announced in October, the Certification Authority (CA) for Apple Push Notification service (APNs) is changing. APNs will update the server certificates in sandbox on January 20, 2025, and in production on February 24, 2025. To continue using APNs without interruption, you’ll need to update your application’s Trust Store to include the new server certificate: SHA-2 Root : USERTrust RSA Certification Authority certificate....

To ensure a smooth transition and avoid push notification delivery failures, please make sure that both old and new server certificates are included in the Trust Store before the cut-off date for each of your application servers that connect to sandbox and production. At this time, you don’t need to update the APNs SSL provider certificates issued to you by Apple.

Do we need to do anytinh on our site if we are using Firebase Messaging?

**Comment by asad-mobiledev** [comment]
See this for Firebase Cloud Messaging: FCM Users https://forums.developer.apple.com/forums/thread/770316?utm_source=chatgpt.com

**Comment by samptn** [comment]
We are having the same in our case also. We are using firebase for our push notification. Is there any changes needed from our side. Can anyone please help?

**Reply by petrnymsa** [reply]
answered here 

https://github.com/firebase/flutterfire/issues/16884
